generic/tests/cfg/timerclock.cfg

```diff
@@ -10,3 +10,6 @@
             type = tsc
         - hwclock:
             type = hwclock
+            date_pattern = "Sat *Feb *2 *03:04:.. 1980"
+            RHEL.8:
+                date_pattern = "1980-02-02 *03:04:"
```

Comment:
Better have RHEL.8 one as default, therefore you don't need append later on versions in the further.

Reply:
Updated.